src/nycdb/datasets/dobjobs.yml

```diff
@@ -3,109 +3,200 @@ files:
+      JOBS1NO: integer
+      TotalConstructionFloorArea: integer
+      WithdrawalFlag: smallint
+      SignoffDate: date
+      SpecialActionStatus: char(1)
+      SpecialActionDate: date
+      BuildingClass: text
+      JobNoGoodCount: smallint
+      GISLATITUDE: numeric
+      GISLONGITUDE: numeric
+      GISCOUNCILDISTRICT: smallint
+      GISCENSUSTRACT: text
+      GISNTANAME: text
+      GISBIN: char(7)
+      bbl: char(10)
+      id: 'serial PRIMARY KEY'
+  -
+    table_name: dob_now_jobs
+    fields:
+      JobFilingNumber: text
+      FilingStatus: text
+      HouseNo: text
+      StreetName: text
+      Borough: text
+      Block: integer
+      LOT: integer
+      Bin: char(7)
+      CommmunityBoard: text
+      WorkonFloor: text
+      AptCondoNos: text
+      ApplicantProfessionalTitle: text
+      ApplicantLicense: text
+      ApplicantFirstName: text
+      ApplicantsMiddleInitial: text
+      ApplicantLastName: text
+      OwnersBusinessName: text
+      OwnersStreetName: text
+      City: text
+      State: text
+      Zip: char(5)
+      FilingRepresentativeFirstName: text
+      FilingRepresentativeMiddleInitial: numeric
+      FilingRepresentativeLastName: text
+      FilingRepresentativeBusinessName: text
+      FilingRepresentativeStreetName: text
+      FilingRepresentativeCity: text
+      FilingRepresentativeState: text
+      FilingRepresentativeZip: text
+      SprinklerWorkType: bool
+      PlumbingWorkType: bool
+      InitialCost: numeric
+      TotalConstructionFloorArea: numeric
+      ReviewBuildingCode: text
+      LittleE: text
+      UnmappedCCOStreet: text
+      RequestLegalization: text
+      IncludesPermanentRemoval: text
+      InCompliancewithNYCECC: bool
+      ExemptfromNYCECC: bool
+      BuildingType: text
+      ExistingStories: numeric
+      ExistingHeight: numeric
+      ExistingDwellingUnits: numeric
+      ProposedNoofStories: numeric
+      ProposedHeight: numeric
+      ProposedDwellingUnits: numeric
+      SpecialInspectionRequirement: text
+      SpecialInspectionAgencyNumber: text
+      ProgressInspectionRequirement: text
+      Built1informationvalue: text
+      Built2informationvalue: text
+      Built2Ainformationvalue: text
+      Built2Binformationvalue: text
+      Standpipe: bool
+      Antenna: bool
+      CurbCut: bool
+      Sign: bool
+      Fence: bool
+      Scaffold: bool
+      Shed: bool
+      FilingDate: date
+      CurrentStatusDate: date
+      PermitIssueDate: date
+      BoilerEquipmentWorkType: bool
+      EarthWorkWorkType: bool
+      FoundationWorkType: bool
+      GeneralConstructionWorkType: bool
+      MechanicalSystemsWorkType: bool
+      PlaceofAssemblyWorkType: bool
+      ProtectionMechanicalMethodsWorkType: bool
+      SidewalkShedWorkType: bool
+      StructuralWorkType: bool
+      SupportofExcavationWorkType: bool
+      TemporaryPlaceofAssemblyWorkType: bool
+      JobType: text
+      bbl: char(10)
+    skip:
+      - Latitude
```

Comment:
out of curiosity, why skip these? I noticed they're blank in all the rows I checked, are they always blank?

Reply:
Yeah so I was skipping them just because they are always blank. I don't know why they are - they might just not be running the geocoding process like they are supposed to before uploading to open data. I filed a ticket with open data asking about this. So yeah it's probably better to just leave them in and then if/when they are populated again we'd get them.

Reply:
also note that pluto_latest has these columns by bbl, so they can be added with a join as needed

Reply:
oh right good point. Ok so for now I'll just leave them out.